rbac/management/role/definitions/approval.json

```diff
@@ -0,0 +1,140 @@
+      "description": "An Approval administrator role that grants create, read, update and destroy permissions",
+      "access": [
+        {
+          "permission": "approval:templates:read",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "approval:workflows:create",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "approval:workflows:read",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "approval:workflows:update",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "approval:workflows:destroy",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "approval:requests:read",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "approval:stages:read",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "approval:actions:read",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "approval:actions:create",
+          "resourceDefinitions": [
+            {
+              "attributeFilter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Approval Approver",
+      "system": true,
+      "version": 1,
```

Comment:
Need to make this version `2` til I get a change in to add this role to existing accounts.